.gitignore

```diff
@@ -1,5 +1,6 @@
 # config file
 config.js
+samples
```

Comment:
tests belong in the `tests` folder. :D

i'm not a fan of blacklisting "special" directories only because they have code that you're trying to hide.